common/changes/@microsoft/rush/chao-pnpm-sync_2024-02-21-20-27.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Turn on usePnpmSyncForInjectedDependencies, and refactor install-test-workspace",
```

```suggestion
      "comment": "",
```

That change isn't meaningful to consumers